FYDP_ModV1/src/version1.java

```diff
@@ -30,1674 +30,1669 @@
+			for (int k = 0; k < teachingCohort; k++) {
+				phys[k] = cplex.linearNumExpr();
+				for (int j = 0; j < n2; j++) {
+					for (int t = 0; t < n4; t++) {
+						phys[k].addTerm(lengtht[t], x[5][j][k][t]);
+					}
+				}
 			}
-		}
-	}
 
-for(int k = 0; k<teachingCohort;k++){
-	cplex.addGe(sci1[k], 80);
-	}
+			for (int k = 0; k < teachingCohort; k++) {
+				cplex.addGe(phys[k], 80);
+			}
 
-//constraint 11, art
-IloLinearNumExpr[] art = new IloLinearNumExpr[teachingCohort];
+			//constraint 14, french for applicable classes
+			IloLinearNumExpr[] french2 = new IloLinearNumExpr[frenchNum];
 
-for(int k = 0; k<teachingCohort; k++){
-	art[k] = cplex.linearNumExpr();
-	for(int j=0;j<n2;j++){
-		for(int t=0;t<n4;t++){
-			art[k].addTerm(lengtht[t], x[3][j][k][t]);
+			for (int k = 0; k < frenchNum; k++) {
+				french2[k] = cplex.linearNumExpr();
+				for (int j = 0; j < frenchTeach; j++) {
+					for (int t = 0; t < n4; t++) {
+						french2[k].addTerm(lengtht[t], x[6][j + frenchTeachlb][k + frenchCohortlb][t]);
+					}
+				}
 			}
-		}
-	}
 
-for(int k = 0;k<teachingCohort; k++){
-	cplex.addGe(art[k], 40);
-}
+			for (int k = 0; k < frenchNum; k++) {
+				cplex.addEq(french2[k], 200);
+			}
 
-//constraint 12, social studies
-IloLinearNumExpr [] soc = new IloLinearNumExpr[teachingCohort];
+			//constraint 14 pt2- primary cant have french
+			IloLinearNumExpr[] noFrench = new IloLinearNumExpr[primary];
 
-for(int k=0; k<teachingCohort;k++){
-	soc[k] = cplex.linearNumExpr();
-	for(int j=0; j<n2;j++){
-		for(int t=0;t<n4;t++){
-			soc[k].addTerm(lengtht[t], x[4][j][k][t]);
+			for (int k = 0; k < primary; k++) {
+				noFrench[k] = cplex.linearNumExpr();
+				for (int j = 0; j < n2; j++) {
+					for (int t = 0; t < n4; t++) {
+						noFrench[k].addTerm(lengtht[t], x[6][j][k][t]);
+					}
+				}
 			}
-		}
-	}
 
-for(int k = 0;k<teachingCohort;k++){
-	cplex.addGe(soc[k],80);
-	}
+			for (int k = 0; k < primary; k++) {
+				cplex.addEq(noFrench[k], 0);
+			}
 
-//constraint 13, phys-ed
-IloLinearNumExpr [] phys = new IloLinearNumExpr[teachingCohort];
+			//constraint 20, music
+			IloLinearNumExpr[] music = new IloLinearNumExpr[teachingCohort];
 
-for(int k=0;k<teachingCohort;k++){
-	phys[k] = cplex.linearNumExpr();
-	for(int j=0;j<n2;j++){
-		for(int t=0;t<n4;t++){
-			phys[k].addTerm(lengtht[t], x[5][j][k][t]);
+			for (int k = 0; k < teachingCohort; k++) {
+				music[k] = cplex.linearNumExpr();
+				for (int j = 0; j < n2; j++) {
+					for (int t = 0; t < n4; t++) {
+						music[k].addTerm(lengtht[t], x[7][j][k][t]);
+					}
+				}
 			}
-		}
-	}
 
-for(int k=0;k<teachingCohort;k++){
-	cplex.addGe(phys[k], 80);
-	}
+			for (int k = 0; k < teachingCohort; k++) {
+				cplex.addGe(music[k], 40);
+			}
 
-//constraint 14, french for applicable classes
-IloLinearNumExpr [] french2 = new IloLinearNumExpr[frenchNum];
+			//constraint 21, drama
+			IloLinearNumExpr[] drama = new IloLinearNumExpr[teachingCohort];
 
-for(int k =0; k<frenchNum;k++){
```

Comment:
quick question: What was the reason for rearranging the constraints?

Reply:
what do you mean by rearranging? The only change was formatting the indents so the open and closed braces were at the same level. I used control shift f or something like that. none of those lines were actually removed, github just picked up changes becuase the indents were all changed. 20 and 21 have always been in on thsoe lines because they were created after the others (adding in music and drama) but they made more sense after all of the other subject constriants

Reply:
The git differential page is making it look like the constraints are being rearranged in their order(more than indents in the earlier part of the file). I moved to reviewing it comparing the entire files because I could not follow the git differential page after a while

Reply:
yea i looked again and the git differential page made the changes look a little wack